service-commands/src/setup.js

```diff
@@ -321,6 +325,14 @@ const allUp = async ({ numCreatorNodes = 4 }) => {
       SetupCommand.REGISTER,
       { ...options, retries: 2 }
     ],
+    [
+      Service.USER_METADATA_NODE,
+      SetupCommand.UNSET_SHELL_ENV
```

Comment:
whats unsetShellEnv for? this is to clean up any env vars from the previous run?

Reply:
yeah we overwrite envvars for each CN run and the problem is currently if you try to down-up a CN after full e2e setup it is broken bc the envvars have been overwritten. this just fixes that

Reply:
resolving per offline